v0.1/index.md

```diff
@@ -69,7 +69,130 @@ Track_ID |  Segment_ID
 
 ### Gap-closing
 
-### Merge/split events
+A gap closing event occurs when an object of interest (a cell) disappear for one frame and then reappears a bit further (next frame or more).
+
+This case can be illustrated as follows:
+
+![4](images/Gap_Closing.png){:class="img-responsive"}
+
+The detected object is lost at frame 3, and then reappears at frame 4.
+In this case, the linking algorithm will create two segments (the yellow and the blue lines in the image).
+
+The **objects table** is:
+
+Object_ID  |  Frame  |  X |  Y
+-----------|---------|----|-----
+1          |    1    |    |  
+2          |    2    |    |  
+3          |    4    |    |
+4          |    5    |    |
+
+The **segments table** is:
+
+Segment_ID |  Object_ID
+-----------|------------
+ 1         |    1
+ 1         |    2
+ 2         |    3
+ 2         |    4
+
+ and the corresponding **tracks table**:
```

Comment:
Good idea - tracks in this specification are represented in tabular data. Better to state it indeed.